forumdirectory/forumdirectory.php

```diff
@@ -56,6 +57,7 @@ function forumdirectory_post(App $a)
 	if (!empty($_POST['search'])) {
 		$a->data['search'] = $_POST['search'];
 	}
+	Logger::log(print_r($_POST, true));
```

Comment:
Please use one of the new `Logger::(error|warning|notice|info|debug)` methods for new logger calls. This looks like a `Logger::debug()` call.

Reply:
This would be a `Logger::data()` call :-) But it shouldn't be there. I forgot to remove it

Reply:
There's no `DATA` nor `TRACE` level anymore, the most verbose is `debug` according to [PSR-3](https://www.php-fig.org/psr/psr-3/)

Reply:
BTW is it somewhere documented which old level corresponds to which  new level?

Reply:
Yes sir:
https://github.com/friendica/friendica/blob/a3152a96c2d1b473f806e287d2e06b2ad4e165ed/src/Core/Logger.php#L16-L39

On the left are the legacy logger levels, on the right are the corresponding PSR-3 compliant levels.